src/ray/object_manager/plasma/store.h

```diff
@@ -224,7 +224,11 @@ class PlasmaStore {
     int64_t num_bytes_in_use =
         static_cast<int64_t>(num_bytes_in_use_ - num_bytes_unsealed_);
     RAY_CHECK(PlasmaAllocator::GetFootprintLimit() >= num_bytes_in_use);
+    size_t max_reservation =
+        RayConfig::instance().maximum_object_store_reservation_for_pulled_objects() *
+        PlasmaAllocator::GetFootprintLimit();
```

Comment:
Instead of "adjusting" the reservable memory here, why not do the multiplication downstream in the module that cares about enforcing the limit?

Reply:
Sure, I don't really have a strong opinion where we do it.